modules/wallpapers/ReadMe.txt

```diff
@@ -24,4 +27,4 @@ To include the module in your build folder, run the configure script with the fl
 
 # For UNIX users
 
-    ./configure.sh -DENABLE_WALLPAPERS=1
\ No newline at end of file
+    ./configure.sh -DENABLE_WALLPAPERS=1
```

Comment:
What is changed here? :thinking:

Reply:
not my changes; merged from master. elaborate the issue?

Reply:
looks like just a whitespace change; i could revert it if you like

Reply:
Please do it. It would be also nice to squash everything into a single commit.